rpm/product.spec

```diff
@@ -203,17 +208,26 @@ then
   # Run genesisInstall
   echo "Running Genesis Install script"
   runuser -l "$genesis_user" -c 'genesisInstall'
-
+  install_error_code=$(echo $?)
+  if [[ $install_error_code != 0 ]]; then
+  	  echo "Genesis $genesis_user genesisInstall has failed at $(date)" >> "$LOG"
+	  exit $install_error_code
+  fi
+  echo "Genesis $genesis_user genesisInstall finished at $(date)" >> "$LOG"
   # Run Remap
   echo "Running Remap"
   runuser -l "$genesis_user" -c 'echo y | remap --commit --force'
   remap_error_code=$(echo $?)
-  if [[ remap_error_code != 0 ]]; then
-	  exit remap_error_code
+  if [[ $remap_error_code != 0 ]]; then
+  	  echo "Genesis $genesis_user remap has failed at $(date)" >> "$LOG"
+	  exit $remap_error_code
   fi
+  echo "Genesis $genesis_user RPM install finished at $(date)" >> "$LOG"
 else
   echo "/tmp/genesis_install is absent or run_exec has been defined in /tmp/genesis_install.conf as: $(sed -n 's/^run_exec=\(.*\)/\1/p' < /tmp/genesis_install.conf)"
+  echo "/tmp/genesis_install is absent or run_exec has been defined in /tmp/genesis_install.conf as: $(sed -n 's/^run_exec=\(.*\)/\1/p' < /tmp/genesis_install.conf)" >> "$LOG"
```

Comment:
I think there's ways to output to the terminal and write to a log at the same time. https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file